SmartDeviceLinkTests/DevAPISpecs/SDLMenuManagerSpec.m

```diff
@@ -257,17 +262,131 @@ @interface SDLMenuManager()
             });
         });
 
-        describe(@"updating when a menu already exists", ^{
+        describe(@"updating when a menu already exists with dynamic updates on", ^{
```

Comment:
Not tested in this spec is:

1. Uploading an image should send the cells again with the artworks (in either dynamic or static updates).
2. `sendSubMenuUpdates:newKeptCells:atIndex:` is mostly untested.

Reply:
fixed